### PR TITLE
Paragraph highlight deletion

### PR DESCRIPTION
**Pull request by viktor992**
Considered the case where a comment related to a whole highlighted paragraph is removed.

### CODE REVIEW

File: client/components/editor.vue [hunk 1]
```diff
@@ -636,35 +636,46 @@
...
-              return y !== commentHighlightKey;
-            });
-            otherKeys = otherKeys || [];
-            if (otherKeys.length < highlightKeys.length) {
-              if (mode === 'after') {
-                highlightNodes.push({pos, otherKeys});
-                pos = this.$editorView.state.doc.resolve(pos.pos + posNode.nodeSize);
-                posNode = pos.nodeAfter;
-                posMarks = pos.nodeAfter.marks;
+        let highlightMark = posNode.marks.find((x) => {
```
Comment (mitar) [inline client/components/editor.vue]: It still fails here. `posNode` might be `null`.

Comment (mitar) [reply client/components/editor.vue]: And it is, in my case, when highlight is on the full parapgraph.

**Comment by mitar** [comment]
In fact this does not work. :-(

**Comment by viktor992** [comment]
Created a new pull request (https://github.com/peer/doc/pull/137) for this bug.

**Pull request merged**